cmd/sealer/cmd/cluster/apply.go

```diff
@@ -127,12 +127,13 @@ func NewApplyCmd() *cobra.Command {
 					return applyClusterWithNew(cf, applyMode, imageEngine, imageSpec)
```

Comment:
applyClusterFileWithNew

Reply:
in fact, it‘s just the part relevant to the cluster.

And as for what you said about the app, that's really unreasonable. And I have added a comment `TODO: decouple the cluster installation and application installation`, and we should fix it in a separate PR.